plugins/scaffolder-backend/src/scaffolder/stages/prepare/file.ts

```diff
@@ -17,7 +17,7 @@ import fs from 'fs-extra';
 import path from 'path';
 import os from 'os';
 import { TemplateEntityV1alpha1 } from '@backstage/catalog-model';
-import { parseLocationAnnotation } from './helpers';
+import { parseLocationAnnotation } from '../helpers';
```

Comment:
Helper functions are now used by both preparers and publishers, so they moved a directory up.